ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts

```diff
@@ -869,28 +869,28 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { effectId: ['D71', 'DAF'] },
       condition: Conditions.targetIsYou(),
       delaySeconds: (_data, matches) => parseFloat(matches.duration) - 8.75,
-      infoText: (data, matches, output) => {
+      alertText: (data, matches, output) => {
         const regression = matches.effectId === 'DAF' ? 'local' : 'remote';
         const defamation = data.defamationColor;
         if (defamation === undefined)
           return;
-        if (regression === 'remote') {
-          const color = defamation === 'red' ? output['blue']!() : output['red']!();
-          return output.nearTether!({ color: color });
-        }
+
+        const defamationTowerColor = defamation === 'red' ? output.red!() : output.blue!();
+        const stackTowerColor = defamation === 'red' ? output.blue!() : output.red!();
+        if (regression === 'remote')
+          return output.farTether!({ color: stackTowerColor });
 
         if (parseFloat(matches.duration) < 80)
-          return output.farTether!({ color: output[defamation]!() });
+          return output.nearTether!({ color: defamationTowerColor });
 
-        const color = defamation === 'red' ? output['blue']!() : output['red']!();
-        return output.finalTowerFar!({ color: color });
+        return output.finalTowerFar!({ color: stackTowerColor });
       },
       outputStrings: {
-        nearTether: {
+        farTether: {
           en: 'Stack by ${color} Tower',
         },
-        farTether: {
-          en: 'Get ${color} Defamation',
+        nearTether: {
+          en: 'Outside ${color} Towers',
         },
         finalTowerFar: {
```

Comment:
This should be changed to finalTowerNear.

Reply:
I'll add that mechanically, this tether group I think can be anywhere but inside a tower as they will have already been immune to defamation, rots, and the stack debuff but most strats I think will just have them be between the non-defamation tower to free up space and explode asap.

Reply:
Thanks for the review, whoops I missed that.